frontend/src/pages/DashboardPage/ControlButtons/ControlButtons.jsx

```diff
@@ -57,7 +58,7 @@ const ControlButtons = ({ onDownload }) => {
   }, []);
 
   const handleImportModels = () => {
-    fileInputRef1.current.click();
+    setShowModal(true);  // Open the modal when button is clicked
```

Comment:
Can this be implemented for upload data also?

Reply:
Yes we can implement for upload data but can you specify the conditions we want to put on the bootstrap modal.

Reply:
We will look into that as the upload dataset is quite straightforward (csv)

Reply:
Will implement this also in my next pr

Reply:
Ok